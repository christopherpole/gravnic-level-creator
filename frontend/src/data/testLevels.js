import { MOVE_DOWN, MOVE_UP } from 'gravnic-game';

export default [
  {
    id: '1',
    name: 'Test level 1',
    solution: [MOVE_DOWN, MOVE_UP],
    tiles: [
      {
        position: 0,
        selectedTileId: '1',
      },
      {
        position: 1,
        selectedTileId: '1',
      },
      {
        position: 2,
        selectedTileId: '1',
      },
      {
        position: 3,
        selectedTileId: '1',
      },
      {
        position: 4,
        selectedTileId: '1',
      },
      {
        position: 5,
        selectedTileId: '1',
      },
      {
        position: 6,
        selectedTileId: '1',
      },
      {
        position: 7,
        selectedTileId: '1',
      },
      {
        position: 8,
        selectedTileId: '1',
      },
      {
        position: 9,
        selectedTileId: '1',
      },
      {
        position: 10,
        selectedTileId: '1',
      },
      {
        position: 11,
        selectedTileId: '1',
      },
      {
        position: 12,
        selectedTileId: '1',
      },
      {
        position: 13,
        selectedTileId: '1',
      },
      {
        position: 14,
        selectedTileId: '1',
      },
      {
        position: 15,
        selectedTileId: '1',
      },
      {
        position: 16,
        selectedTileId: '1',
      },
      {
        position: 17,
        selectedTileId: '1',
      },
      {
        position: 18,
        selectedTileId: '1',
      },
      {
        position: 19,
        selectedTileId: '1',
      },
      {
        position: 20,
        selectedTileId: '1',
      },
      {
        position: 21,
        selectedTileId: '1',
      },
      {
        position: 22,
        selectedTileId: '1',
      },
      {
        position: 23,
        selectedTileId: '2',
      },
      {
        position: 24,
        selectedTileId: '1',
      },
      {
        position: 25,
        selectedTileId: '1',
      },
      {
        position: 26,
        selectedTileId: '1',
      },
      {
        position: 27,
        selectedTileId: '1',
      },
      {
        position: 28,
        selectedTileId: '1',
      },
      {
        position: 29,
        selectedTileId: '1',
      },
      {
        position: 30,
        selectedTileId: '1',
      },
      {
        position: 31,
        selectedTileId: '1',
      },
      {
        position: 32,
        selectedTileId: '2',
      },
      {
        position: 33,
        selectedTileId: '2',
      },
      {
        position: 34,
        selectedTileId: '2',
      },
      {
        position: 35,
        selectedTileId: '2',
      },
      {
        position: 36,
        selectedTileId: '2',
      },
      {
        position: 37,
        selectedTileId: '2',
      },
      {
        position: 38,
        selectedTileId: '1',
      },
      {
        position: 39,
        selectedTileId: '1',
      },
      {
        position: 40,
        selectedTileId: '1',
      },
      {
        position: 41,
        selectedTileId: '1',
      },
      {
        position: 42,
        selectedTileId: '2',
      },
      {
        position: 43,
        selectedTileId: '2',
      },
      {
        position: 44,
        selectedTileId: '2',
      },
      {
        position: 45,
        selectedTileId: '2',
      },
      {
        position: 46,
        selectedTileId: '2',
      },
      {
        position: 47,
        selectedTileId: '2',
      },
      {
        position: 48,
        selectedTileId: '4',
      },
      {
        position: 49,
        selectedTileId: '1',
      },
      {
        position: 50,
        selectedTileId: '1',
      },
      {
        position: 51,
        selectedTileId: '1',
      },
      {
        position: 52,
        selectedTileId: '2',
      },
      {
        position: 53,
        selectedTileId: '2',
      },
      {
        position: 54,
        selectedTileId: '2',
      },
      {
        position: 55,
        selectedTileId: '2',
      },
      {
        position: 56,
        selectedTileId: '2',
      },
      {
        position: 57,
        selectedTileId: '1',
      },
      {
        position: 58,
        selectedTileId: '1',
      },
      {
        position: 59,
        selectedTileId: '1',
      },
      {
        position: 60,
        selectedTileId: '1',
      },
      {
        position: 61,
        selectedTileId: '1',
      },
      {
        position: 62,
        selectedTileId: '2',
      },
      {
        position: 63,
        selectedTileId: '2',
      },
      {
        position: 64,
        selectedTileId: '2',
      },
      {
        position: 65,
        selectedTileId: '2',
      },
      {
        position: 66,
        selectedTileId: '1',
      },
      {
        position: 67,
        selectedTileId: '1',
      },
      {
        position: 68,
        selectedTileId: '1',
      },
      {
        position: 69,
        selectedTileId: '1',
      },
      {
        position: 70,
        selectedTileId: '1',
      },
      {
        position: 71,
        selectedTileId: '1',
      },
      {
        position: 72,
        selectedTileId: '1',
      },
      {
        position: 73,
        selectedTileId: '1',
      },
      {
        position: 74,
        selectedTileId: '1',
      },
      {
        position: 75,
        selectedTileId: '1',
      },
      {
        position: 76,
        selectedTileId: '1',
      },
      {
        position: 77,
        selectedTileId: '1',
      },
      {
        position: 78,
        selectedTileId: '1',
      },
      {
        position: 79,
        selectedTileId: '1',
      },
      {
        position: 80,
        selectedTileId: '1',
      },
      {
        position: 81,
        selectedTileId: '1',
      },
      {
        position: 82,
        selectedTileId: '1',
      },
      {
        position: 83,
        selectedTileId: '1',
      },
      {
        position: 84,
        selectedTileId: '1',
      },
      {
        position: 85,
        selectedTileId: '1',
      },
      {
        position: 86,
        selectedTileId: '1',
      },
      {
        position: 87,
        selectedTileId: '1',
      },
      {
        position: 88,
        selectedTileId: '1',
      },
      {
        position: 89,
        selectedTileId: '1',
      },
      {
        position: 90,
        selectedTileId: '1',
      },
      {
        position: 91,
        selectedTileId: '1',
      },
      {
        position: 92,
        selectedTileId: '1',
      },
      {
        position: 93,
        selectedTileId: '1',
      },
      {
        position: 94,
        selectedTileId: '1',
      },
      {
        position: 95,
        selectedTileId: '1',
      },
      {
        position: 96,
        selectedTileId: '1',
      },
      {
        position: 97,
        selectedTileId: '1',
      },
      {
        position: 98,
        selectedTileId: '1',
      },
      {
        position: 99,
        selectedTileId: '1',
      },
    ],
    stars: [1, 2, 3],
    position: 1,
  },
  {
    id: '2',
    name: 'Test level 2',
    solution: [MOVE_UP],
    tiles: [
      {
        position: 0,
        selectedTileId: '1',
      },
      {
        position: 1,
        selectedTileId: '1',
      },
      {
        position: 2,
        selectedTileId: '1',
      },
      {
        position: 3,
        selectedTileId: '1',
      },
      {
        position: 4,
        selectedTileId: '1',
      },
      {
        position: 5,
        selectedTileId: '1',
      },
      {
        position: 6,
        selectedTileId: '1',
      },
      {
        position: 7,
        selectedTileId: '1',
      },
      {
        position: 8,
        selectedTileId: '1',
      },
      {
        position: 9,
        selectedTileId: '1',
      },
      {
        position: 10,
        selectedTileId: '1',
      },
      {
        position: 11,
        selectedTileId: '1',
      },
      {
        position: 12,
        selectedTileId: '1',
      },
      {
        position: 13,
        selectedTileId: '1',
      },
      {
        position: 14,
        selectedTileId: '1',
      },
      {
        position: 15,
        selectedTileId: '1',
      },
      {
        position: 16,
        selectedTileId: '1',
      },
      {
        position: 17,
        selectedTileId: '1',
      },
      {
        position: 18,
        selectedTileId: '1',
      },
      {
        position: 19,
        selectedTileId: '1',
      },
      {
        position: 20,
        selectedTileId: '1',
      },
      {
        position: 21,
        selectedTileId: '1',
      },
      {
        position: 22,
        selectedTileId: '1',
      },
      {
        position: 23,
        selectedTileId: '2',
      },
      {
        position: 24,
        selectedTileId: '1',
      },
      {
        position: 25,
        selectedTileId: '1',
      },
      {
        position: 26,
        selectedTileId: '1',
      },
      {
        position: 27,
        selectedTileId: '1',
      },
      {
        position: 28,
        selectedTileId: '1',
      },
      {
        position: 29,
        selectedTileId: '1',
      },
      {
        position: 30,
        selectedTileId: '1',
      },
      {
        position: 31,
        selectedTileId: '1',
      },
      {
        position: 32,
        selectedTileId: '2',
      },
      {
        position: 33,
        selectedTileId: '2',
      },
      {
        position: 34,
        selectedTileId: '2',
      },
      {
        position: 35,
        selectedTileId: '2',
      },
      {
        position: 36,
        selectedTileId: '2',
      },
      {
        position: 37,
        selectedTileId: '2',
      },
      {
        position: 38,
        selectedTileId: '1',
      },
      {
        position: 39,
        selectedTileId: '1',
      },
      {
        position: 40,
        selectedTileId: '1',
      },
      {
        position: 41,
        selectedTileId: '1',
      },
      {
        position: 42,
        selectedTileId: '2',
      },
      {
        position: 43,
        selectedTileId: '2',
      },
      {
        position: 44,
        selectedTileId: '2',
      },
      {
        position: 45,
        selectedTileId: '2',
      },
      {
        position: 46,
        selectedTileId: '2',
      },
      {
        position: 47,
        selectedTileId: '4',
      },
      {
        position: 48,
        selectedTileId: '2',
      },
      {
        position: 49,
        selectedTileId: '1',
      },
      {
        position: 50,
        selectedTileId: '2',
      },
      {
        position: 51,
        selectedTileId: '2',
      },
      {
        position: 52,
        selectedTileId: '2',
      },
      {
        position: 53,
        selectedTileId: '2',
      },
      {
        position: 54,
        selectedTileId: '2',
      },
      {
        position: 55,
        selectedTileId: '2',
      },
      {
        position: 56,
        selectedTileId: '2',
      },
      {
        position: 57,
        selectedTileId: '1',
      },
      {
        position: 58,
        selectedTileId: '1',
      },
      {
        position: 59,
        selectedTileId: '1',
      },
      {
        position: 60,
        selectedTileId: '1',
      },
      {
        position: 61,
        selectedTileId: '1',
      },
      {
        position: 62,
        selectedTileId: '2',
      },
      {
        position: 63,
        selectedTileId: '2',
      },
      {
        position: 64,
        selectedTileId: '2',
      },
      {
        position: 65,
        selectedTileId: '2',
      },
      {
        position: 66,
        selectedTileId: '1',
      },
      {
        position: 67,
        selectedTileId: '1',
      },
      {
        position: 68,
        selectedTileId: '1',
      },
      {
        position: 69,
        selectedTileId: '1',
      },
      {
        position: 70,
        selectedTileId: '1',
      },
      {
        position: 71,
        selectedTileId: '1',
      },
      {
        position: 72,
        selectedTileId: '1',
      },
      {
        position: 73,
        selectedTileId: '1',
      },
      {
        position: 74,
        selectedTileId: '1',
      },
      {
        position: 75,
        selectedTileId: '1',
      },
      {
        position: 76,
        selectedTileId: '1',
      },
      {
        position: 77,
        selectedTileId: '1',
      },
      {
        position: 78,
        selectedTileId: '1',
      },
      {
        position: 79,
        selectedTileId: '1',
      },
      {
        position: 80,
        selectedTileId: '1',
      },
      {
        position: 81,
        selectedTileId: '1',
      },
      {
        position: 82,
        selectedTileId: '1',
      },
      {
        position: 83,
        selectedTileId: '1',
      },
      {
        position: 84,
        selectedTileId: '1',
      },
      {
        position: 85,
        selectedTileId: '1',
      },
      {
        position: 86,
        selectedTileId: '1',
      },
      {
        position: 87,
        selectedTileId: '1',
      },
      {
        position: 88,
        selectedTileId: '1',
      },
      {
        position: 89,
        selectedTileId: '1',
      },
      {
        position: 90,
        selectedTileId: '1',
      },
      {
        position: 91,
        selectedTileId: '1',
      },
      {
        position: 92,
        selectedTileId: '1',
      },
      {
        position: 93,
        selectedTileId: '1',
      },
      {
        position: 94,
        selectedTileId: '1',
      },
      {
        position: 95,
        selectedTileId: '1',
      },
      {
        position: 96,
        selectedTileId: '1',
      },
      {
        position: 97,
        selectedTileId: '1',
      },
      {
        position: 98,
        selectedTileId: '1',
      },
      {
        position: 99,
        selectedTileId: '1',
      },
    ],
    stars: [1, 2, 3],
    position: 2,
  },
  {
    id: '3',
    name: 'Test level 3',
    solution: [],
    tiles: [
      {
        position: 0,
        selectedTileId: '1',
      },
      {
        position: 1,
        selectedTileId: '1',
      },
      {
        position: 2,
        selectedTileId: '1',
      },
      {
        position: 3,
        selectedTileId: '1',
      },
      {
        position: 4,
        selectedTileId: '1',
      },
      {
        position: 5,
        selectedTileId: '1',
      },
      {
        position: 6,
        selectedTileId: '1',
      },
      {
        position: 7,
        selectedTileId: '1',
      },
      {
        position: 8,
        selectedTileId: '1',
      },
      {
        position: 9,
        selectedTileId: '1',
      },
      {
        position: 10,
        selectedTileId: '1',
      },
      {
        position: 11,
        selectedTileId: '1',
      },
      {
        position: 12,
        selectedTileId: '1',
      },
      {
        position: 13,
        selectedTileId: '1',
      },
      {
        position: 14,
        selectedTileId: '1',
      },
      {
        position: 15,
        selectedTileId: '1',
      },
      {
        position: 16,
        selectedTileId: '1',
      },
      {
        position: 17,
        selectedTileId: '1',
      },
      {
        position: 18,
        selectedTileId: '1',
      },
      {
        position: 19,
        selectedTileId: '1',
      },
      {
        position: 20,
        selectedTileId: '1',
      },
      {
        position: 21,
        selectedTileId: '1',
      },
      {
        position: 22,
        selectedTileId: '1',
      },
      {
        position: 23,
        selectedTileId: '2',
      },
      {
        position: 24,
        selectedTileId: '1',
      },
      {
        position: 25,
        selectedTileId: '1',
      },
      {
        position: 26,
        selectedTileId: '1',
      },
      {
        position: 27,
        selectedTileId: '1',
      },
      {
        position: 28,
        selectedTileId: '1',
      },
      {
        position: 29,
        selectedTileId: '1',
      },
      {
        position: 30,
        selectedTileId: '1',
      },
      {
        position: 31,
        selectedTileId: '1',
      },
      {
        position: 32,
        selectedTileId: '2',
      },
      {
        position: 33,
        selectedTileId: '2',
      },
      {
        position: 34,
        selectedTileId: '2',
      },
      {
        position: 35,
        selectedTileId: '2',
      },
      {
        position: 36,
        selectedTileId: '2',
      },
      {
        position: 37,
        selectedTileId: '2',
      },
      {
        position: 38,
        selectedTileId: '1',
      },
      {
        position: 39,
        selectedTileId: '1',
      },
      {
        position: 40,
        selectedTileId: '1',
      },
      {
        position: 41,
        selectedTileId: '1',
      },
      {
        position: 42,
        selectedTileId: '2',
      },
      {
        position: 43,
        selectedTileId: '2',
      },
      {
        position: 44,
        selectedTileId: '2',
      },
      {
        position: 45,
        selectedTileId: '4',
      },
      {
        position: 46,
        selectedTileId: '2',
      },
      {
        position: 47,
        selectedTileId: '2',
      },
      {
        position: 48,
        selectedTileId: '2',
      },
      {
        position: 49,
        selectedTileId: '2',
      },
      {
        position: 50,
        selectedTileId: '2',
      },
      {
        position: 51,
        selectedTileId: '1',
      },
      {
        position: 52,
        selectedTileId: '1',
      },
      {
        position: 53,
        selectedTileId: '2',
      },
      {
        position: 54,
        selectedTileId: '2',
      },
      {
        position: 55,
        selectedTileId: '2',
      },
      {
        position: 56,
        selectedTileId: '2',
      },
      {
        position: 57,
        selectedTileId: '1',
      },
      {
        position: 58,
        selectedTileId: '1',
      },
      {
        position: 59,
        selectedTileId: '1',
      },
      {
        position: 60,
        selectedTileId: '1',
      },
      {
        position: 61,
        selectedTileId: '1',
      },
      {
        position: 62,
        selectedTileId: '2',
      },
      {
        position: 63,
        selectedTileId: '2',
      },
      {
        position: 64,
        selectedTileId: '2',
      },
      {
        position: 65,
        selectedTileId: '2',
      },
      {
        position: 66,
        selectedTileId: '1',
      },
      {
        position: 67,
        selectedTileId: '1',
      },
      {
        position: 68,
        selectedTileId: '1',
      },
      {
        position: 69,
        selectedTileId: '1',
      },
      {
        position: 70,
        selectedTileId: '1',
      },
      {
        position: 71,
        selectedTileId: '1',
      },
      {
        position: 72,
        selectedTileId: '1',
      },
      {
        position: 73,
        selectedTileId: '1',
      },
      {
        position: 74,
        selectedTileId: '1',
      },
      {
        position: 75,
        selectedTileId: '1',
      },
      {
        position: 76,
        selectedTileId: '1',
      },
      {
        position: 77,
        selectedTileId: '1',
      },
      {
        position: 78,
        selectedTileId: '1',
      },
      {
        position: 79,
        selectedTileId: '1',
      },
      {
        position: 80,
        selectedTileId: '1',
      },
      {
        position: 81,
        selectedTileId: '1',
      },
      {
        position: 82,
        selectedTileId: '1',
      },
      {
        position: 83,
        selectedTileId: '1',
      },
      {
        position: 84,
        selectedTileId: '1',
      },
      {
        position: 85,
        selectedTileId: '1',
      },
      {
        position: 86,
        selectedTileId: '1',
      },
      {
        position: 87,
        selectedTileId: '1',
      },
      {
        position: 88,
        selectedTileId: '1',
      },
      {
        position: 89,
        selectedTileId: '1',
      },
      {
        position: 90,
        selectedTileId: '1',
      },
      {
        position: 91,
        selectedTileId: '1',
      },
      {
        position: 92,
        selectedTileId: '1',
      },
      {
        position: 93,
        selectedTileId: '1',
      },
      {
        position: 94,
        selectedTileId: '1',
      },
      {
        position: 95,
        selectedTileId: '1',
      },
      {
        position: 96,
        selectedTileId: '1',
      },
      {
        position: 97,
        selectedTileId: '1',
      },
      {
        position: 98,
        selectedTileId: '1',
      },
      {
        position: 99,
        selectedTileId: '1',
      },
    ],
    stars: [1, 2, 3],
    position: 3,
  },
];
