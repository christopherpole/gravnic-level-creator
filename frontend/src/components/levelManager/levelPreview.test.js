import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import LevelPreview from './levelPreview';
import testLevels from '../../data/testLevels';

configure({ adapter: new Adapter() });

describe('<LevelPreview />', () => {
  let props;

  beforeEach(() => {
    props = {
      tiles: testLevels[0].tiles,
    };
  });

  it('Renders without exploding', () => {
    const level = shallow(<LevelPreview {...props} />);

    expect(level).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const level = shallow(<LevelPreview {...props} />);

    expect(toJson(level)).toMatchSnapshot();
  });
});
