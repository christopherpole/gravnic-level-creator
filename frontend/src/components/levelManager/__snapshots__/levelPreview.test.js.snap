// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The level preview Matches the current snapshot 1`] = `
<styled.div
  className="level-preview"
>
  <styled.div>
    <styled.div
      className="preview-tile"
      key="0"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="1"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="2"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="3"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="4"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="5"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="6"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="7"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="8"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="9"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="10"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="11"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="12"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="13"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="14"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="15"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="16"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="17"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="18"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="19"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="20"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="21"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="22"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="23"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="24"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="25"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="26"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="27"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="28"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="29"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="30"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="31"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="32"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="33"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="34"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="35"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="36"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="37"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="38"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="39"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="40"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="41"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="42"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="43"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="44"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="45"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="46"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="47"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="48"
    >
      <Tile
        tileId={2}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="49"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="50"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="51"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="52"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="53"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="54"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="55"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="56"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="57"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="58"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="59"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="60"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="61"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="62"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="63"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="64"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="65"
    >
      <Tile
        tileId={1}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="66"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="67"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="68"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="69"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="70"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="71"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="72"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="73"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="74"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="75"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="76"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="77"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="78"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="79"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="80"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="81"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="82"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="83"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="84"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="85"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="86"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="87"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="88"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="89"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="90"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="91"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="92"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="93"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="94"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="95"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="96"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="97"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="98"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
    <styled.div
      className="preview-tile"
      key="99"
    >
      <Tile
        tileId={0}
      />
    </styled.div>
  </styled.div>
</styled.div>
`;
